wrappers/python/tests/TestPickle.py

```diff
@@ -39,16 +45,23 @@ def test_deepcopy(self):
         system2 = copy.deepcopy(system)
         integrator2 = copy.deepcopy(integrator)
         state2 = copy.deepcopy(state)
-        
+
         str_state = pickle.dumps(state)
         str_integrator = pickle.dumps(integrator)
-        
+
         state3 = pickle.loads(str_state)
         context.setState(state3)
 
-
         del context, integrator
 
+        # Check deep copy of each force.
+        forces = [ system.getForce(index) for index in range(system.getNumForces()) ]
+        for force in forces:
+            force_copy = copy.deepcopy(force)
+            # Check class name is same.
+            self.assertEqual(force.__class__.__name__, force_copy.__class__.__name__)
+            # TODO: Check to make sure all force data properly copied?
```

Comment:
You can do that by serializing each of them and comparing the XML.  That would be a good test to do for the system, state, and integrator as well (above).

    self.assertEqual(XMLSerializer.serialize(force), XMLSerializer.serialize(force_copy))